portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,25 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Home</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
+    <div class="topnav">
+      <a class="active" href="#home">Aaron Lopes</a>
+      <a href="projects.html">projects</a>
+      <a href="socials.html">socials</a>
+      <a href="misc.html">misc</a>
```

Comment:
Since you added projects.html and socials.html, it might be worthwhile to add a misc.html file too so you don't get a 404 when you click on it.